apps/prairielearn/src/cron/autoFinishExams.ts

```diff
@@ -38,6 +38,8 @@ export async function run() {
     const close = !!assessment_instance.open;
     // Override any submission or grading rate limits.
     const overrideGradeRate = true;
+    // Override real-time grading disabled checks.
+    const overrideRealTimeGradingDisabled = true;
```

Comment:
nit: Should we prefer a positive boolean for this (`overrideRealTimeGradingEnabled = false`)? It doesn't really matter though.

Reply:
I went this way for consistency with `overrideGradeRate`: the default is to respect the config, but you specifically turn on the overrides.